### PR TITLE
docs(publisher): Add an example of streaming contract events. DS-64

### PR DESCRIPTION
**Pull request by lostman**

### CODE REVIEW

**Review by 0xterminator — DISMISSED** [review]
Good stuff!

File: examples/streams/src/main.rs [hunk 1]
```diff
@@ -124,3 +151,83 @@ async fn stream_transactions(
...
+    contract_id: ContractId,
+) -> anyhow::Result<()> {
+    let mut receipt_stream = fuel_streams::Stream::<Receipt>::new(client).await;
+
+    receipt_stream.with_filter(
+        ReceiptsBurnSubject::new().with_contract_id(Some(contract_id.into())),
+    );
+    receipt_stream.with_filter(
+        ReceiptsCallSubject::new().with_from(Some(contract_id.into())),
+    );
```
Comment (Jurshsmith) [inline examples/streams/src/main.rs:184]: This might be better from a readability standpoint:

```suggestion
    let mut receipt_stream = fuel_streams::Stream::<Receipt>::new(client).await;
    let receipt_stream = receipt_stream.with_filter(
        ReceiptsBurnSubject::new().with_contract_id(Some(contract_id.into())),
    ).with_filter(
        ReceiptsCallSubject::new().with_from(Some(contract_id.into())),
    ).
    // other with_filters come here
```

**Review by pedronauck — COMMENTED** [review]
How about transactions and other resources from a specific contract id? You can use the `by_id` subject to add examples too.

File: examples/publisher/Cargo.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 [package]
-name = "fuel-streams-publisher-examples"
+name = "fuels-example-publisher"
```
Comment (Jurshsmith) [inline examples/publisher/Cargo.toml:2]: Sorry, this is a bit confusing. Why this change? Are the examples for `fuels` or `fuel-streams`?

File: examples/streams/Cargo.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "fuel-streams-examples"
+name = "fuels-example-streams"
```
Comment (Jurshsmith) [inline examples/streams/Cargo.toml:2]: same here

**Review by pedronauck — DISMISSED** [review]
